openldap/assets/configuration/spec.yaml

```
@@ -0,0 +1,82 @@
+name: OpenLDAP
+files:
+- name: openldap.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: url
+      description: |
+        Full URL of your ldap server. Use `ldaps` or `ldap` as the scheme to
+        use TLS or not, or `ldapi` to connect to a UNIX socket.
+      required: True
+      value:
+        example: ldaps://localhost:636
+        type: string
+    - name: username
+      description: The Distinguished Name of the user that can read the monitor database.
+      value:
+        type: string
+        example: <USER_DISTINGUISHED_NAME>
+    - name: password
+      description: Password associated with `username`
+      value:
+        type: string
+    - name: ssl_verify
+      description: Whether or not to perform server certificate validation.
+      value:
+        type: boolean
+        example: true
+    - name: ssl_key
+      description: Path to the client private key file, in PEM format.
+      value:
+        type: string
+        example: <KEY_PEM_PATH>
+    - name: ssl_cert
+      description: |
+        Path to the client certificate file in PEM format.
+        If you don't specify a key, it's taken from the certificate file.
```

```suggestion
        If you don't specify a key, the path is taken from the certificate file.
```
at least, I think that's what "it" means here. Please clarify. if not.